error_parity/classifiers.py

```diff
@@ -279,11 +279,17 @@ def find_weights_given_two_points(
                 f"should be 1!"
             )
 
-        if not all(np.isclose(target_point, all_weights @ all_points)):
+        if not all(np.isclose(target_point, all_weights @ all_points, atol=1e-5)):
             raise RuntimeError(
                 f"Triangulation of target point failed. "
                 f"Target was {target_point}; got {all_weights @ all_points}."
             )
+        
+        if (all_weights < 0).any():
```

Comment:
Did you ever encounter a scenario where any weights were negative? And if so, could you post a code example?